docs/feed.md

```diff
@@ -21,6 +21,18 @@ permalink: /feed.xml
 
 The `permalink` value is the location of the generated feed.
 
+## Create a collection of posts
+
+The feed will include all pages in a collection called 'post'.
+
+You can create this by adding some code to your `eleventy.config.js`:
+
+```js
+eleventyConfig.addCollection('post', (collection) => {
+  return collection.getFilteredByGlob('app/posts/*.md')
+})
+```
+
```

Comment:
I wonder if it’s better to be a little more explicit, and call this collection `feed` (the Atom feed might not contain what the author considers to be ‘posts’).

Also, I wonder if it’s possible to create a default collection, that could be overridden. Maybe a `feed` collection that gets all Markdown files with an explicit date (`page.data.date` I think)? Maybe that’s something to think about for later.

Reply:
@frankieroberto Thoughts on the above?

Reply:
Sorry, missed this. I like `feed` as a default collection name.

Can you remind me if there’s a way for the page itself to set the collection name to override the default collection referenced in the template? (That way you could have multiple feeds in a project if needed)

Reply:
Without testing, you could do something like this in the feed template:

```njk
{% set collection = data.collection or "feed" %}

{%- for post in collections[collection] | reverse %}
  …
{%- endfor %}
```

Then, in the front matter, a user could do the following:

```yaml
---
eleventyExcludeFromCollections: true
layout: feed
permalink: /feed.xml
collection: kittens
---
```

Reply:
@paulrobertlloyd I'm wondering now whether it makes sense to use the pagination options, in the same way as the `collection` layout?

Then you could do:

```yaml
eleventyExcludeFromCollections: true
layout: feed
permalink: /feed.xml
pagination:
  data: collections.post
  size: 20
  reverse: true
```

And in the layout itself:

```nunjucks
{%- for item in pagination.items %}
```

Reply:
Oh yes, yes, yes! This is precisely the right approach.